lib/samly/sp_handler.ex

```diff
@@ -119,6 +119,13 @@ defmodule Samly.SPHandler do
     get_session(conn, "target_url") || "/"
   end
 
+  defp put_session_new(conn, key, value) do
```

Comment:
I saw `put_session_new` above and wondered how I'd missed that in the past because it looks really useful. Turns out I didn't miss it 😄